src/Picqer/Carriers/SendCloud/Connection.php

```diff
@@ -112,12 +112,16 @@ public function apiUrl()
      * Perform a GET request
      * @param string $url
      * @param array $params
+     * @param string $auth
      * @return array
      * @throws SendCloudApiException
      */
-    public function get($url, $params = [])
+    public function get($url, $params = [], $auth = false)
     {
         try {
+            if ($auth) {
+                $params[$auth] = $this->apiKey;
```

Comment:
In that case, maybe it makes more sense to do `if (!is_null($auth)) {` instead and set the default value for `$auth` to `null` instead. Seems a bit more readable than mixing boolean and strings IMHO.